dev/io.openliberty.microprofile.reactive.messaging.internal_fat.common/src/com/ibm/ws/microprofile/reactive/messaging/fat/kafka/framework/KafkaTestClient.java

```diff
@@ -72,6 +72,8 @@ public KafkaTestClient(String kafkaBootstrap) {
      * <p>
      * The connection properties are properties common to every consumer or producer connecting to this broker.
      *
+     * Once and instance of KafkaReader is no longer needed. It must be closed either via KafkaTestClient.cleanup() or KafkaReader.close()
+     *
```

Comment:
Use `<p>` between paragraphs otherwise IDEs will lose the line breaks when displaying the doc for the method in a popup.

"Once and instance" -> "Once an instance"